### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by HajimeKawahara**
Release branch for v1.0.0. We plan to merge the release branch to the master one on 10th Dec 2021, unless we find any severe problem.

### CODE REVIEW

**Comment by HajimeKawahara** [comment]
@ykawashima 

During the tests in the tutorial [here](http://secondearths.sakura.ne.jp/exojax/userguide/ql.html) , I have met such errors about vaex/hdf conversion when the second H2O file is converted (in this test, we use two H2O 04200-04300 and 04300-04000).
Have you ever seen such errors before?

```sh
Background atmosphere:  H2
Reading transition file
Note: Couldn't find the hdf5 format. We convert data to the hdf5 format. After the second time, it will become much faster.
Traceback (most recent call last):
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/exojax-1.0.0-py3.8.egg/exojax/spec/exomolapi.py", line 133, in read_trans
    dat = vaex.from_csv(transf,compression="bz2",sep="\s+",names=("i_upper","i_lower","A","nu_lines"),convert=True)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/__init__.py", line 534, in from_csv
    vaex.convert.convert_csv(
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/convert.py", line 51, in convert_csv
    cached_output(*args, **kwargs)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/cache.py", line 315, in call
    value = callable(*args, **kwargs)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/convert.py", line 50, in cached_output
    _from_csv_convert_and_read(path_input, path_output=path_output, fs_options=fs_options_input, fs=fs_input, **kwargs)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/convert.py", line 66, in _from_csv_convert_and_read
    for i, df_pandas in enumerate(csv_reader):
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1024, in __next__
    return self.get_chunk()
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1074, in get_chunk
    return self.read(nrows=size)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1047, in read
    index, columns, col_dict = self._engine.read(nrows)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 223, in read
    chunks = self._reader.read_low_memory(nrows)
  File "pandas/_libs/parsers.pyx", line 813, in pandas._libs.parsers.TextReader.read_low_memory
  File "pandas/_libs/parsers.pyx", line 857, in pandas._libs.parsers.TextReader._read_rows
  File "pandas/_libs/parsers.pyx", line 843, in pandas._libs.parsers.TextReader._tokenize_rows
  File "pandas/_libs/parsers.pyx", line 1917, in pandas._libs.parsers.raise_parser_error
  File "/home/kawahara/anaconda3/lib/python3.8/bz2.py", line 195, in read1
    return self._buffer.read1(size)
  File "/home/kawahara/anaconda3/lib/python3.8/_compression.py", line 68, in readinto
    data = self.read(len(byte_view))
  File "/home/kawahara/anaconda3/lib/python3.8/_compression.py", line 99, in read
    raise EOFError("Compressed file ended before the "
EOFError: Compressed file ended before the end-of-stream marker was reached

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/exojax-1.0.0-py3.8.egg/exojax/spec/autospec.py", line 324, in addmol
    axs=AutoXS(self.nus,database,molecules,crit=crit,xsmode=self.xsmode, autogridconv=self.autogridconv)        
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/exojax-1.0.0-py3.8.egg/exojax/spec/autospec.py", line 57, in __init__
    self.init_database()
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/exojax-1.0.0-py3.8.egg/exojax/spec/autospec.py", line 65, in init_database
    self.mdb=moldb.MdbExomol(molpath,nurange=[self.nus[0],self.nus[-1]],broadf=self.broadf,crit=self.crit)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/exojax-1.0.0-py3.8.egg/exojax/spec/moldb.py", line 175, in __init__
    trans=exomolapi.read_trans(trans_file)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/exojax-1.0.0-py3.8.egg/exojax/spec/exomolapi.py", line 135, in read_trans
    dat = vaex.read_csv(transf,sep="\s+",names=("i_upper","i_lower","A","nu_lines"),convert=True)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/__init__.py", line 560, in read_csv
    return from_csv(filepath_or_buffer, **kwargs)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/__init__.py", line 534, in from_csv
    vaex.convert.convert_csv(
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/convert.py", line 51, in convert_csv
    cached_output(*args, **kwargs)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/cache.py", line 315, in call
    value = callable(*args, **kwargs)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/convert.py", line 50, in cached_output
    _from_csv_convert_and_read(path_input, path_output=path_output, fs_options=fs_options_input, fs=fs_input, **kwargs)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/vaex/convert.py", line 66, in _from_csv_convert_and_read
    for i, df_pandas in enumerate(csv_reader):
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1024, in __next__
    return self.get_chunk()
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1074, in get_chunk
    return self.read(nrows=size)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1047, in read
    index, columns, col_dict = self._engine.read(nrows)
  File "/home/kawahara/anaconda3/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 223, in read
    chunks = self._reader.read_low_memory(nrows)
  File "pandas/_libs/parsers.pyx", line 813, in pandas._libs.parsers.TextReader.read_low_memory
  File "pandas/_libs/parsers.pyx", line 857, in pandas._libs.parsers.TextReader._read_rows
  File "pandas/_libs/parsers.pyx", line 843, in pandas._libs.parsers.TextReader._tokenize_rows
  File "pandas/_libs/parsers.pyx", line 1917, in pandas._libs.parsers.raise_parser_error
  File "/home/kawahara/anaconda3/lib/python3.8/bz2.py", line 195, in read1
    return self._buffer.read1(size)
  File "/home/kawahara/anaconda3/lib/python3.8/_compression.py", line 68, in readinto
    data = self.read(len(byte_view))
  File "/home/kawahara/anaconda3/lib/python3.8/_compression.py", line 99, in read
    raise EOFError("Compressed file ended before the "
EOFError: Compressed file ended before the end-of-stream marker was reached

```

**Comment by HajimeKawahara** [comment]
The above error was due to the download error of trans.gz2. @ykawashima Thanks for the suggestion. 
I removed trans.bz2, redo, then found it was fine.

**Comment by ykawashima** [comment]
I am glad to hear that it worked!

File: documents/userguide/installation.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ In addition, you may need cuDNN for `response.ipgauss2 <../exojax/exojax.spec.ht
 References
 =================
 
-**ExoJAX Paper I**:  Kawahara, Kawashima, Masuda, Crossfield, Parker, van den Bekerom (2021) accepted by ApJS: `arXiv:2105.14782 <http://arxiv.org/abs/2105.14782>`_
+|:green_circle:| **ExoJAX Paper I**:  Kawahara, Kawashima, Masuda, Crossfield, Parker, van den Bekerom (2021) accepted by ApJS: `arXiv:2105.14782 <http://arxiv.org/abs/2105.14782>`_
```
Comment (erwanp) [inline documents/userguide/installation.rst:50]: small typo
```suggestion
|:green_circle:| **ExoJAX Paper I**:  Kawahara, Kawashima, Masuda, Crossfield, Pannier, van den Bekerom (2021) accepted by ApJS: `arXiv:2105.14782 <http://arxiv.org/abs/2105.14782>`_
```

Comment (HajimeKawahara) [reply documents/userguide/installation.rst]: sorry. corrected!

**Review by astrostevanus — APPROVED** [review]
I have tested all of the basic tutorial in the documentation and I can confirm that everything works well.

**Comment by HajimeKawahara** [comment]
Thanks!

**Comment by ykawashima** [comment]
@HajimeKawahara @chonma0ctopus This is just to share the information, but when I tried to calculate cross-section using MODIT and [HiroyukiIshikawa.4214450.gz](http://secondearths.sakura.ne.jp/exojax/data/), I encountered the following error.
```
/home/kawashima/anaconda3/lib/python3.7/site-packages/exojax-1.0.0-py3.7.egg/exojax/spec/dit.py:208: RuntimeWarning: divide by zero encountered in log10
  lxmin=np.log10(np.min(x))
Traceback (most recent call last):
  File "MODIT_cs_VALD.py", line 53, in <module>
    ngammaL_grid=ditgrid(ngammaL)
  File "/home/kawashima/anaconda3/lib/python3.7/site-packages/exojax-1.0.0-py3.7.egg/exojax/spec/modit.py", line 235, in ditgrid
    return ditgrid_(x,res,adopt)
  File "/home/kawashima/anaconda3/lib/python3.7/site-packages/exojax-1.0.0-py3.7.egg/exojax/spec/dit.py", line 211, in ditgrid
    Ng=int(dlog/res)+2
OverflowError: cannot convert float infinity to integer
```
It seems this happens because gammaL of some lines are somehow zero, resulting in infinity [dlog](https://github.com/HajimeKawahara/exojax/blob/8d7dac2a882b72e36d5be904617c8c8f03e56c84/src/exojax/spec/dit.py#L209). If this can happen for the other VALD line lists too, it might be better to exclude the lines with zero gammaL to enable the MODIT computations.

**Comment by HajimeKawahara** [comment]
@ykawashima Thank you for finding this issue. I am not sure how we should deal with the line having zero gamma in VALD3.
@chonma0ctopus Do you have an opinion on that?

**Review by ykawashima — APPROVED** [review]
I have confirmed that the code works for cross-section and spectrum calculations for any combination of LPF/MODIT and HITRAN/HITEMP/ExoMol/VALD except for a small issue I have mentioned above for the MODIT calculation using VALD.

**Comment by HajimeKawahara** [comment]
@ykawashima Thank you! It looks the practical issue is that we cannot compute the dit grid (LSD) when there exists zero gamma. So, I just exclude zero gamma to make the DIT grid. Can you check if this implementation works (formally)?

**Comment by HajimeKawahara** [comment]
I just added Warning for zero or negative gamma when setting ditgrid.

**Pull request merged**